spec/lib/monitoring/metrics/policy_metrics_spec.rb

```diff
@@ -81,13 +81,13 @@ def headers_with_auth(payload)
     end
 
     it 'updates the registry' do
-      resources_before = registry.get(:conjur_resource_count).get(labels: { kind: 'group' })
-      roles_before = registry.get(:conjur_role_count).get(labels: { kind: 'group' })
+      resources_before = registry.get(@resource_metric.metric_name).get(labels: { kind: 'group' })
+      roles_before = registry.get(@role_metric.metric_name).get(labels: { kind: 'group' })
 
       post(policies_url, env: headers_with_auth('[!group test]'))
 
-      resources_after = registry.get(:conjur_resource_count).get(labels: { kind: 'group' })
-      roles_after = registry.get(:conjur_role_count).get(labels: { kind: 'group' })
+      resources_after = registry.get(@resource_metric.metric_name).get(labels: { kind: 'group' })
+      roles_after = registry.get(@role_metric.metric_name).get(labels: { kind: 'group' })
 
```

Comment:
The object definition for each metric exposes the metric name as an attribute. It seemed better to load it from the object here rather than hardcode the value in case it changes down the road.